psalm.xml.dist

```diff
@@ -53,6 +53,7 @@
                 <referencedClass name="Symfony\Bridge\Doctrine\Attribute\MapEntity"/>
                 <referencedClass name="Symfony\Component\VarExporter\LazyGhostTrait"/>
                 <referencedClass name="Symfony\Component\VarExporter\LazyObjectInterface"/>
+                <referencedClass name="Symfony\Bridge\Doctrine\SchemaListener\DoctrineDbalCacheAdapterSchemaListener"/>
```

Comment:
Replace with `@psalm-suppress`. We add exclusion rules to global config only when inline suppression does not work.

Reply:
ok !